app/containers/Avatar/index.tsx

```diff
@@ -21,19 +25,24 @@ const AvatarContainer = ({
 	onPress,
 	getCustomEmoji,
 	isStatic,
-	rid
-}: IAvatar): React.ReactElement => {
+	rid,
+	handleEdit,
+	isUserProfile
+}: IAvatar & { isUserProfile?: boolean; handleEdit?: () => void }): React.ReactElement => {
```

Comment:
It looks like you should update IAvatar

Reply:
Actually, I created a interface IAvatarContainer